src/reducers/navbar/index.js

```diff
@@ -0,0 +1,26 @@
+import * as actionTypes from '../../actions/navbar/types';
+
+export const getNavbarReducer = (state = { navbar: [] }, action) => {
```

Comment:
Yo lo guardaria el estado inicial en una variable para evitar confusiones.
```suggestion
const initialState - {
    navbar: []
}
export const getNavbarReducer = (state = initialState, action) => {
```

Reply:
perfecto